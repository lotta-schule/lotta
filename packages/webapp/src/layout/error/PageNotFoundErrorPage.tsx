import { memo } from 'react';

import { FullErrorPage } from './FullErrorPage';

export const PageNotFoundErrorPage = memo(() => {
  return (
    <FullErrorPage
      title={'Diese Seite existiert nicht'}
      imageUrl={'/RoadSignNotFound.svg'}
    >
      <p>
        Die Seite konnte nicht gefunden werden. Entweder ist die Adresse falsch,
        oder die Seite wurde entfernt.
      </p>
    </FullErrorPage>
  );
});
PageNotFoundErrorPage.displayName = 'PageNotFoundErrorPage';
