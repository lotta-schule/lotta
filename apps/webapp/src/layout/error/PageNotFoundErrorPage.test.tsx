import { render } from '@testing-library/react';
import { PageNotFoundErrorPage } from './PageNotFoundErrorPage';

describe('PageNotFoundErrorPage', () => {
  it('should render the PageNotFoundErrorPage component', () => {
    const screen = render(<PageNotFoundErrorPage />);
    expect(screen.container).toMatchSnapshot();
  });
});
