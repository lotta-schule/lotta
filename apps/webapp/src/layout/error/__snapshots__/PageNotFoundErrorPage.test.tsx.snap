// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`PageNotFoundErrorPage > should render the PageNotFoundErrorPage component 1`] = `
<div>
  <section
    class="_error_42c9b3"
  >
    <h1>
      Diese Seite existiert nicht
    </h1>
    <div>
      <img
        alt="Diese Seite existiert nicht"
        role="presentation"
        src="/RoadSignNotFound.svg"
      />
      <p>
        Die Seite konnte nicht gefunden werden. Entweder ist die Adresse falsch, oder die Seite wurde entfernt.
      </p>
    </div>
  </section>
</div>
`;
